pdfgen.h

```diff
@@ -158,6 +158,22 @@ struct pdf_path_operation {
  */
 #define PDF_TRANSPARENT (uint32_t)(0xffu << 24)
 
+/*!
```

Comment:
These #defines are really only used inside pdfgen.c, not by the users of it. Can you move them from pdfgen.h to pdfgen.c?